server/components/filters/index.js

```diff
@@ -475,6 +475,15 @@
       return `${hours > 0 ? hours + (hours > 1 ? ' hours ' : ' hour ') : ''}${mins > 0 ? mins + (mins > 1 ? ' minutes' : ' minute') : ''}`
     },
 
+    hoursStringToNoHours: function(time) {
```

Comment:
because this calculates a duration maybe we rename it `timeToDuration` or `hoursToDuration`? 😄

Reply:
Makes sense, changed to timeToDuration